include/crypto/riscv_arch.h

```diff
@@ -61,4 +61,7 @@ static const size_t kRISCVNumCaps =
 #define RISCV_HAS_ZBKB_AND_ZKND_AND_ZKNE() (RISCV_HAS_ZBKB() && RISCV_HAS_ZKND() && RISCV_HAS_ZKNE())
 #define RISCV_HAS_ZKND_AND_ZKNE() (RISCV_HAS_ZKND() && RISCV_HAS_ZKNE())
 
+/* Get the size of a vector register in bits (VLEN). */
```

Comment:
Minor: is it legal to call it when RISCV_HAS_V() is false? Do we want to guarantee that the result is 0 then?

Reply:
Is not documented accordingly.